raiden_contracts/tests/test_channel_settle.py

```diff
@@ -75,7 +70,6 @@ def test_settle_no_bp_success(
     ).transact({'from': A})
 
 
-@pytest.mark.parametrize('channel_test_values', channel_settle_test_values)
 def test_settle_channel_state(
```

Comment:
I left this test as a basic state test with 1 balance proof pair, following the pattern for all other tests.
Moved the more comprehensive tests for old balance proofs with various values in `test_channel_settle_unlock_state.py`, also testing the balances at the end of the channel lifecycle - so not only for `settleChannel`, but also for `unlock`.